### PR TITLE
Add bmc modal

### PR DESCRIPTION
**Pull request by mareklibra**
Depends on:
- [x] #604  DownloadIsoModal

### CODE REVIEW

**Comment by mareklibra** [comment]
---

**Comment by nirfarkas** [comment]
@mareklibra Where is this supposed to live?

**Comment by mareklibra** [comment]
@nirfarkas , that will be consumed by the ACM. Recently WIP to include that in the dynamic-cim plugin (part of the OCP). There is no plan to show that dialog in the AI UI.

File: src/components/bmc/utils.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { AddBmcValues } from './types';
+
+export const getBareMetalHostCredentialsSecret = (
```
Comment (rawagner) [inline src/components/bmc/utils.ts:3]: shouldnt this live in `dynamic-cim` ?

Comment (mareklibra) [reply src/components/bmc/utils.ts:3]: I think those methods are generic enough to be used by various consumers which will need that k8s resource composition in their flows.

**Review by jtomasek — APPROVED** [review]

**Review by rawagner — APPROVED** [review]

**Pull request merged**